src/collection-browser.ts

```diff
@@ -1125,6 +1125,14 @@ export class CollectionBrowser
     );
   }
 
+  private emitTabChanged(tabId: string) {
+    this.dispatchEvent(
+      new CustomEvent<String>('tabChanged', {
+        detail: tabId,
+      })
+    );
+  }
+
```

Comment:
Suggestion: Let's make it so collection browser doesn't need to have knowledge of its parent component's tabs. Maybe we just emit an `emptyResults` event with no args, and let the parent handle that as appropriate?

Reply:
I used custom event with args to make is reusable if future if we want to communicate with offfshoot later to switch between tabs. that way, we can simply send tabId to collection-browser to switch.

Reply:
+1 to emptyResults bc tab is not direct child of collection browser, it's slotted 👍 please do update to emptyResults